weblogic-azure-aks/src/main/arm/scripts/common.sh

```diff
@@ -31,3 +33,7 @@ export optUninstallMaxTry=5 # Max attempts to wait for the operator uninstalled
 export optUninstallInterval=10
 
 export wlsContainerName="weblogic-server"
+export wdtDownloadURL="https://github.com/oracle/weblogic-deploy-tooling/releases/download/release-1.9.17/weblogic-deploy.zip"
+export witDownloadURL="https://github.com/oracle/weblogic-image-tool/releases/download/release-1.9.16/imagetool.zip"
+export wlsPostgresqlDriverUrl="https://jdbc.postgresql.org/download/postgresql-42.3.6.jar"
+export wlsMSSQLDriverUrl="https://repo.maven.apache.org/maven2/com/microsoft/sqlserver/mssql-jdbc/10.2.1.jre8/mssql-jdbc-10.2.1.jre8.jar"
```

Comment:
Maybe we can do something like:
```
export wlsMSSQLDriverUrl="https://repo.maven.apache.org/maven2/com/microsoft/sqlserver/mssql-jdbc/10.2.1.jre8/${constMSSQLDriverName}"
```
Not sure if it's better, because the download path `/10.2.1.jre8/` seems not that friendly

Download link from GitHub also follows this pattern: https://github.com/microsoft/mssql-jdbc/releases/download/v10.2.1/mssql-jdbc-10.2.1.jre8.jar

Reply:
Good catching!

Reply:
But I think it's reasonable to keep the default path, as this folder has all the drivers, it helps us with the location if someday we want to include another version.

So I will keep the url link.